certified-connectors/ShareEffect/apiDefinition.swagger.json

```diff
@@ -207,7 +225,13 @@
                 },
                 "Termid": {
                   "type": "string",
-                  "description": "Termid"
+                  "description": "Termid",
+                  "title": ""
+                },
+                "Term": {
+                  "type": "string",
+                  "description": "Term",
+                  "title": ""
```

Comment:
@Rens-Braspenning Empty property values can be removed until unless you have a specific reason to keep them.